webui/react/src/components/Transfer.tsx

```diff
@@ -189,15 +182,16 @@ const Transfer: React.FC<Props> = ({
       <div className={css.entries}>
         <div className={css.column}>
           <h2>{sourceListTitle}</h2>
-          <List
-            className={css.listContainer}
-            height={200}
-            innerElementType="ul"
-            itemCount={filteredHiddenEntries.length}
-            itemSize={24}
-            width="100%">
-            {renderHiddenRow}
-          </List>
+          <div className={css.listContainer} ref={rightRef}>
+            <ul>
+              <Virtuoso
+                customScrollParent={rightRef.current ?? undefined}
```

Comment:
q: do you need a `customScrollParent` here? if not, remove from the other list as well

Reply:
didnt need it, so removed it